dev-libs/immer/immer-0.8.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="A library of persistent and immutable data structures written in C++"
+HOMEPAGE="https://sinusoid.es/immer/"
+SRC_URI="https://github.com/arximboldi/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Boost-1.0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+IUSE=""
+#RESTRICT="!test? ( test )"
+SLOT=0
```

Comment:
Please reorder according to `skel.ebuild` and do the same for all other new ebuilds:
```
LICENSE
SLOT
KEYWORDS
IUSE
```

Did you personally test it on all these arches?
KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"

Reply:
I did not test on all of these arches but I think we would have to strip everything besides amd64 from the krita ebuild then.
I don't think that is a good approach, no?

Reply:
It is the **only** approach for new packages introduced to the tree. [1] Keywords must be dropped except for the one you have tested with. Further keywords are added by arch teams as requested in a keywordreq bug.

[1] https://devmanual.gentoo.org/ebuild-maintenance/new-ebuild/index.html#initial-architecture-keywords